SonarRunner.Shim/PropertiesWriter.cs

```diff
@@ -95,7 +95,7 @@ public string Flush()
             return sb.ToString();
         }
 
-        public void WriteSettingsForProject(ProjectInfo project, IEnumerable<string> files, ProjectLanguage projectLanguage, string fxCopReportFilePath, string codeCoverageFilePath)
```

Comment:
The projectLanguage is part of the ProjectInfo so doesn't need to be passed in separately.